### ISSUE TITLE
motion_imitate.py error 

### ISSUE THREAD

**Issue by thelordlugo** [iPERDance/iPERCore#5]
Hi, have any clues? thank you guy so much for your work

<img width="1045" alt="Screen Shot 2020-12-06 at 9 17 19 PM" src="https://user-images.githubusercontent.com/20141834/101302056-9c69b800-3808-11eb-9ff6-6186a85b79c7.png">

**Comment by piaozhx** [comment]
Are you running on your custom video but not demo video from our assets dir? 
it looks like a problem in the process of "pre-processing".

**Comment by thelordlugo** [comment]
> Are you running on your custom video but not demo video from our assets dir?
> it looks like a problem in the process of "pre-processing".

hi, thanks for the reply, this is exactly what I put into the terminal   ` python demo/motion_imitate.py --gpu_ids 0    --image_size 512    --num_source 1      --output_dir "./results"    --assets_dir "./assets"     --model_id   "donald_trump_2"    --src_path   "path?=/data/dev/projects/iPERCore/assets/samples/sources/donald_trump_2/00000.PNG,name?=donald_trump_2"    --ref_path   "path?=/data/dev/projects/iPERCore/assets/samples/references/akun_2.mp4,name?=akun_2,pose_fc?=300"`

**Comment by StevenLiuWen** [comment]
hi, @thelordlugo , it seems that you have not installed the iPERCore successfully. Have you run **python setup.py develop** before you run the motion_imitate.py. If possible, could provide a snapshot of the installation stage?

**Comment by thelordlugo** [comment]
Hi @StevenLiuWen Thanks for the reply, I reinstalled it, it doesn't show that error anymore, now it shows `RuntimeError: nms is not compiled with GPU support`

**Comment by StevenLiuWen** [comment]
Hi @thelordlugo , it seems that there are some problems when install MMCV or mmdet，our preprocessing rely on these two packages.

**Comment by StevenLiuWen** [comment]
hi @thelordlugo ，What's the Cuda version of your system?

**Comment by thelordlugo** [comment]
@StevenLiuWen hi, it's 11.0

**Comment by thelordlugo** [comment]
@StevenLiuWen made an error, it's 11.1

**Comment by StevenLiuWen** [comment]
@thelordlugo , currently we do not have the system with CUDA 11.0+.
you can try the followings:
```Python
pip uninstall mmcv mmcv-full, 
pip install mmcv-full==1.2.0
```

Let us know, if this works, and we will update the setup.py

**Comment by thelordlugo** [comment]
@StevenLiuWen Hi, fixed it, works great now, had to completely uninstall Cuda 11.1 and installed 10.2. Thanks!

**Comment by StevenLiuWen** [comment]
@thelordlugo , It seems that there are some problems in setup.py when installing with CUDA 11.0+. We only test on CUDA 10.1 and CUDA 10.2. We will try to fix it with CUDA 11.0+ later.

**Comment by thelordlugo** [comment]
@StevenLiuWen  awesome! I have one more question if you don't mind, I have another pc at home with 4 Tesla V100's, does it scale well across multiple GPU's? 

Thanks again
